skflow/__init__.py

```diff
@@ -51,11 +51,12 @@ class TensorFlowEstimator(BaseEstimator):
 
     def __init__(self, model_fn, n_classes, tf_master="", batch_size=32, steps=50, optimizer="SGD",
                  learning_rate=0.1, tf_random_seed=42, continue_training=False,
-                 log_device_placement=True):
+                 log_device_placement=True, verbose=1):
```

Comment:
Hmm, still seeing the conflict. Make sure you have the right upstream master `git remote add upstream https://github.com/google/skflow.git`, `git fetch upstream master`, `git merge upstream/master`. (Note that there are different ways to do this). You should see the google/skflow printed out by `git remote -v`.

There should be conflict and you should resolve it, save, commit, and then push again.